mac-install-git-vim-janus

```diff
@@ -1,27 +1,43 @@
 #!/usr/bin/env bash
-# -----------------------------------------------
-# This idempotent macOS Bash script uses
-# Homebrew to install basic dev tools such as
-# vim and git
-# Note that this installs the Janus vim plugins
-# -----------------------------------------------
 
 # Exit script on any errors
 set -e
 
+usage() {
+  cat << EOF
+Usage: $(basename $0) [-h]
+This idempotent macOS Bash script uses Homebrew to install vim and git
+with the Janus vim plugins
+
+OPTIONS:
+  -h  display help
+EOF
+}
+
+while getopts 'h' opt; do
+  case "$opt" in
+    ?|h)
+    usage
+    exit
+      ;;
+  esac
+done
+shift "$(($OPTIND -1))"
+
+# -- MAIN --
 echo "Homebrew installing git and vim with janus extensions:"
 dev_tools=(
   git
   vim
 )
 
 # Homebrew is required
-brew help || \
+brew help > /dev/null || \
```

Comment:
Silences checks. Risk is that it either doesn't work or it yields false positive or negative.